packages/@react-spectrum/actiongroup/docs/ActionGroup.mdx

```diff
@@ -75,7 +75,7 @@ ActionGroups are uncontrolled by default.
 Use `defaultSelectedKeys` to provide a default set of selected items. The value of the selected keys must match the `key` prop of the items.
 
 ```tsx example
-<ActionGroup defaultSelectedKeys={['delete']}>
+<ActionGroup selectionMode="single" defaultSelectedKeys={['delete']}>
```

Comment:
Should we change the options in here to be more real-world? They seem more like actions right now rather than selectable options.